app/Record.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+class Record 
```

Comment:
You're missing a namespace here. Every class should have a namespace based on their path in the folder. So, for example, if Record.php is inside the `app` folder, it's namespace should be `namespace App;` (added at the top of the file). Then whenever you use that class, instead of using `Record` you will use `App\Record`.